watertap/examples/flowsheets/mvc/tests/test_mvc_single_stage.py

```diff
@@ -0,0 +1,341 @@
+        assert value(m.fs.hx_distillate.cold.deltaP[0]) == 7e3
+        assert m.fs.hx_distillate.hot.deltaP[0].is_fixed()
+        assert value(m.fs.hx_distillate.hot.deltaP[0]) == 7e3
+
+        assert m.fs.hx_brine.overall_heat_transfer_coefficient[0].is_fixed()
+        assert value(m.fs.hx_brine.overall_heat_transfer_coefficient[0]) == 2e3
+        assert m.fs.hx_brine.area.is_fixed()
+        assert value(m.fs.hx_brine.area) == 115
+        assert m.fs.hx_brine.cold.deltaP[0].is_fixed()
+        assert value(m.fs.hx_brine.cold.deltaP[0]) == 7e3
+        assert m.fs.hx_brine.hot.deltaP[0].is_fixed()
+        assert value(m.fs.hx_brine.hot.deltaP[0]) == 7e3
+
+        # evaporator
+        assert m.fs.evaporator.outlet_brine.temperature[0].is_fixed()
+        assert value(m.fs.evaporator.outlet_brine.temperature[0]) == 343.15
+        assert m.fs.evaporator.U.is_fixed()
+        assert value(m.fs.evaporator.U) == 3e3
+
+        # compressor
+        assert m.fs.compressor.pressure_ratio.is_fixed()
+        assert value(m.fs.compressor.pressure_ratio) == 1.6
+        assert m.fs.compressor.efficiency.is_fixed()
+        assert value(m.fs.compressor.efficiency) == 0.8
+
+        # 0 TDS in distillate
+        assert (
+            m.fs.tb_distillate.properties_out[0]
+            .flow_mass_phase_comp["Liq", "TDS"]
+            .is_fixed()
+        )
+        assert (
+            value(
+                m.fs.tb_distillate.properties_out[0].flow_mass_phase_comp["Liq", "TDS"]
+            )
+            == 1e-5
+        )
+
+        # Costing
+        assert m.fs.costing.factor_total_investment.is_fixed()
+        assert value(m.fs.costing.factor_total_investment) == 2
+        assert m.fs.costing.heat_exchanger.material_factor_cost.is_fixed()
+        assert value(m.fs.costing.heat_exchanger.material_factor_cost) == 5
+        assert m.fs.costing.evaporator.material_factor_cost.is_fixed()
+        assert value(m.fs.costing.evaporator.material_factor_cost) == 5
+        assert m.fs.costing.compressor.unit_cost.is_fixed()
+        assert value(m.fs.costing.compressor.unit_cost) == 7364
+
+        # Temperature upper bounds
+        assert value(m.fs.evaporator.properties_vapor[0].temperature.ub) == 348.15
+        assert (
+            value(m.fs.compressor.control_volume.properties_out[0].temperature.ub)
+            == 450
+        )
+
+        # check degrees of freedom
+        assert degrees_of_freedom(m) == 1
+
+    @pytest.mark.component
+    def test_initialize_system(self, mvc_single_stage):
+        m = mvc_single_stage
+
+        initialize_system(m)
+
+        # mass flows in evaporator
+        assert value(
+            m.fs.evaporator.properties_vapor[0].flow_mass_phase_comp["Vap", "H2O"]
+        ) == pytest.approx(22.222, rel=1e-3)
+        # assert value(
+        #     m.fs.evaporator.properties_vapor[0].flow_mass_phase_comp["Liq", "H2O"]
+        # ) == pytest.approx(1e-8, rel=1e-3)
```

Comment:
Commented code- you can delete, or if this gave you trouble, change from relative tolerance to absolute tolerance since this is meant to indicate 0, like so:
```suggestion
        #     m.fs.evaporator.properties_vapor[0].flow_mass_phase_comp["Liq", "H2O"]
        # ) == pytest.approx(1e-8, abs=1e-3)
```